.gitignore

```diff
@@ -1,2 +1,3 @@
 build
```

Comment:
Might be worth just using the node .gitignore that Github has [here](https://github.com/github/gitignore/blob/master/Node.gitignore)